### PR TITLE
Translate versioning policy

### PR DESCRIPTION
**Pull request by Oedipus**

### CODE REVIEW

File: content/docs/faq-versioning.md [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
-* When releasing **bug fixes**, we make a **patch release** by changing the **z** number (ex: 15.6.2 to 15.6.3).
+* **Uyumsuz değişiklikleri** yayınlarken **x** değerini güncelleyerek  **ana sürümü** yayınlarız  (Örneğin: 15.6.2'den 16.0.0'a geçiş).
+* **Yeni özellikleri** yayınlarken **y** değerini güncelleyerek **minör sürümü** yayınlarız (Örneğin: 15.6.2'den 15.7.0'a geçiş).
+* **Hata düzeltmelerini** yayınlarken **z** değerini güncelleyerek **yama sürümünü** yayınlarız (Örneğin: 15.6.2'den 15.6.3'a geçiş).
 
-Major releases can also contain new features, and any release can include bug fixes.
+Ana sürümler de yeni özellikler içerebilir, ve herhangi bir yeni sürüm hata düzeltmelerini bulundurabilir.
 
-### Breaking Changes {#breaking-changes}
+### Uyumsuz değişiklikler {#breaking-changes}
```
Comment (alioguzhan) [inline content/docs/faq-versioning.md]: Capitalized `d` in `degisiklikler`

File: content/docs/faq-versioning.md [hunk 2]
```diff
@@ -1,48 +1,49 @@
...
 
-Breaking changes are inconvenient for everyone, so we try to minimize the number of major releases – for example, React 15 was released in April 2016 and React 16 was released in September 2017; React 17 isn't expected until 2019.
+Uyumsuz değişiklikler hepimiz için zahmetli olduğu için ana sürümlerin sayısını olabildiğince az tutmaya çalışıyoruz – örneğin, React 15, Nisan 2016'da, React 16, Eylül 2017'de yayınlanmıştır; React 17'nin 2019'dan önce yayınlanması beklenmemektedir.
 
-Instead, we release new features in minor versions. That means that minor releases are often more interesting and compelling than majors, despite their unassuming name.
+Bunun yerine yeni özellikleri minör sürümlerle yayınlamaktayız. Bu nedenle minör sürümler çoğu zaman ana sürümlerden daha ilgi çekici ve merak uyandırıcı olmaktadır.
 
-### Commitment to Stability {#commitment-to-stability}
+### Stabiliteye Bağlılık {#commitment-to-stability}
```
Comment (alioguzhan) [inline content/docs/faq-versioning.md]: I believe we can translate `stability` as `kararlilik`.

File: content/docs/faq-versioning.md [hunk 3]
```diff
@@ -1,48 +1,49 @@
...
+React kullanan bir milyondan fazla geliştirici, milyonlarca React bileşenini devam ettirmektedir. Sadece Facebook'un kodunda 50.000'den fazla React bileşeni vardır. Bu nedenle React versiyon yükseltemelerini olabildiğince kolay hale getirmemiz gerekmektedir; eğer büyük değişikler yapar ve buna geçiş için bir yol haritası sunmazsak, geliştiriciler eski sürümlerde takılı kalırlar. Geçiş için sunduğumuz çözümleri Facebook'ta kendimiz de test etmekteyiz - eğer 10 kişiden az olan takımımız, 50.000'den fazla bileşeni güncelleyebiliyorsa, React kullanan herkesin bu geçişi kolaylıkla yapabileceğine inanıyoruz. Çoğu zaman  [otomatikleştirilmiş scriptlerle](https://github.com/reactjs/react-codemod) bileşenleri güncelliyoruz, ve sonrasında bunları herkesin kullanması için açık kaynak sürümümüze dahil ediyoruz.
 
-### Gradual Upgrades via Warnings {#gradual-upgrades-via-warnings}
+### Uyarılarla kademeli versiyon geçişi {#gradual-upgrades-via-warnings}
```
Comment (alioguzhan) [inline content/docs/faq-versioning.md]: Capitalize all words and possibly better translation:

> Uyarilarla Kademeli Yukseltmeler

File: content/docs/faq-versioning.md [hunk 4]
```diff
@@ -1,48 +1,49 @@
 ---
 id: faq-versioning
-title: Versioning Policy
+title: Sürümleme Prensipleri
```
Comment (alioguzhan) [inline content/docs/faq-versioning.md]: Suggestion: I believe `Surum Politikasi`  or with your approach `Surumleme Politikasi` is better for this translation.

File: content/docs/faq-versioning.md (hunk 2 above)
Comment (Oedipus) [reply content/docs/faq-versioning.md]: I changed it but although I'm native Turkish speaker, `Kararliliga Baglilik` does not make much sense to me in the given context. 😅  Thanks for the feedback.

**Comment by Oedipus** [comment]
Hi @alioguzhan,
 I applied your suggestions. Thanks for your feedback

File: content/docs/faq-versioning.md (hunk 2 above)
Comment (alioguzhan) [reply content/docs/faq-versioning.md]: :D  I see your point. Stable is translated as `kararlı` in many local articles/projects including Pardus and Debian websites. See the results, please. : [Google - Kararlı Sürüm](https://www.google.com/search?q=kararl%C4%B1+s%C3%BCr%C3%BCm)

That's why I came up with that translation. What about now? :) You can use your own translation though. That was just a suggestion.

Comment (Oedipus) [reply content/docs/faq-versioning.md]: Oh I see, makes much sense now. Thank you.

**Pull request merged**